scripts/attachSessionHelper.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var fs = require('fs'),
+    http = require('http');
+
+var postData = JSON.stringify({'desiredCapabilities': {'browserName': 'chrome'}});
+var options = {
+  hostname: 'localhost',
+  port: 4444,
+  path: '/wd/hub/session',
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/x-www-form-urlencoded',
+    'Content-Length': Buffer.byteLength(postData)
+  }
+};
+var TEST = {};
+TEST.body = '';
+
+var setBody = function(chunk) {
+  TEST.body += chunk.toString();
+  var sessionId = JSON.parse(TEST.body).sessionId;
+  console.log('created new session with id: ' + sessionId);
+  fs.writeFile('spec/attachSessionTemp.js',
```

Comment:
I think it would be cleaner to use Node's [child_process](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) to start up protractor and add a command line flag, e.g. `bin/protractor spec/attachSessionConf.js --sessionId=***`

Then we don't have to mess with the file system.